src/condor_ce_startup

```diff
@@ -4,13 +4,19 @@ fail() {
     RETVAL="$1"
     MSG="$2"
     echo
-    echo "ERROR: $MSG"
+    echo -e "ERROR: $MSG"
     exit $RETVAL
 }
 
 # Source condor-ce environment
 [ -f /usr/share/condor-ce/condor_ce_env_bootstrap ] && . /usr/share/condor-ce/condor_ce_env_bootstrap
 
+# Verify that required dirs are owned by condor:condor (SOFTWARE-2806``)
+required_dirs="/var/run/condor-ce /var/log/condor-ce /var/log/condor-ce/user /var/lib/condor-ce /var/lib/condor-ce/spool
+/var/lib/condor-ce/execute /var/lock/condor-ce /var/lock/condor-ce/user"
+bad_dirs=$(stat --format "%n (%U:%G)" $required_dirs | awk '$2 != "(condor:condor)" {print $0}')
```

Comment:
Another trick is to use `find` for file stat checks, eg:
```
bad_dirs=$(find $required_dirs -maxdepth 0 ! \( -user condor -group condor \) )
```
although the `stat ... | awk ...` solution works even when no `condor` user/group exists.